parser/operator/operator.go

```diff
@@ -20,6 +20,15 @@ func IsBoolean(op string) bool {
 	return op == "and" || op == "or" || op == "&&" || op == "||"
 }
 
+func IsFunc(op string) bool {
```

Comment:
I know we have a PR with this func in separate PR. But for this PR lets rename the func to something like:  `AllowNegatePrefix`

Reply:
Renamed function name as "AllowedNegateSuffix"